drivers/nrf24l01p/include/gnrc_netif_nrf24l01p.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2019 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+/**
+ * @ingroup drivers_nrf24l01p
+ * @ingroup drivers_netdev
+ * @ingroup net_gnrc_netif
+ * @{
+ *
+ * @file
+ * @brief   NRF24L01P adaptation for @ref net_gnrc_netif
+ *
+ * @author  Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author  Peter Kietzmann <peter.kietzmann@haw-hamburg.de>
+ * @author  Fabian Hüßler <fabian.huessler@ovgu.de>
+ */
+#ifndef GNRC_NETIF_NRF24L01P_H
+#define GNRC_NETIF_NRF24L01P_H
+
+#include "net/gnrc/netif.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Default protocol for data that is coming in
+ */
+#ifdef MODULE_GNRC_SIXLOWPAN
```

Comment:
You could use `#if IS_USED(MODULE_GNRC_SIXLOWPAN)` for consistency.

Reply:
Ok